designs/lucy/src/pocket.js

```diff
@@ -51,10 +51,10 @@ export default function (part) {
       .line(points.middle)
       .attr('class', 'path fabric dashed')
 
-    points.logo = points.center.shiftOutwards(points.middle, pocketWidth / 5)
+    points.logo = points.centerLeft.shift(270, pocketWidth / 3)
```

Comment:
Checked in the lab. This may be better with pocketLength rather than pocketWidth so suggesting but reject if you think is fine.
```suggestion
    points.logo = points.centerLeft.shift(270, pocketLength / 3)
```

Reply:
I don't think it makes a big enough difference no <3